build.gradle

```diff
@@ -71,6 +71,19 @@ reportUploader {
 
 apply from: 'dependencies.gradle'
 
+apply from: 'dependency_license.gradle'
+
+// Custom task to run checkLicense in buildSrc, which is not triggered
```

Comment:
Do we have an issue filed to move the buildSrc into a separate project and get rid of this? If not, could we file one please?